src/brouwers/kitreviews/models.py

```diff
@@ -1,17 +1,20 @@
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.core.urlresolvers import reverse
+from django.core.validators import MaxValueValidator, MinValueValidator
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from djchoices import DjangoChoices, ChoiceItem
 
 from brouwers.albums.models import Album
 from brouwers.general.utils import get_username
 
-
 RATING_BASE = 100  # store ratings relative to 100
 RATING_DISPLAY_BASE = 5
 DEFAULT_RATING = 50
 DEFAULT_DIFFICULTY = 3
```

Comment:
In that case, nuke it :-)
On Apr 4, 2016 8:29 AM, "Alex Khomenko" <notifications@github.com> wrote:

> In src/brouwers/kitreviews/models.py
> <https://github.com/modelbrouwers/modelbrouwers/pull/47#discussion_r58332335>
> :
>
> >
> >  from brouwers.albums.models import Album
> >  from brouwers.general.utils import get_username
> >
> > -
> >  RATING_BASE = 100  # store ratings relative to 100
> >  RATING_DISPLAY_BASE = 5
> >  DEFAULT_RATING = 50
> >  DEFAULT_DIFFICULTY = 3
>
> Yeah, I was thinking the same. The DEFAULT_RATING is used afaik, but there
> rest don't seem necessary anymore.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/modelbrouwers/modelbrouwers/pull/47/files/c5219b1fa2e9b039ecf102ec03e4ec19127e646c#r58332335>
>